src/test/java/net/sf/jsqlparser/expression/mysql/SelectVisitorFactory.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
could both of these live in just your test class? (otherwise the name might throw people off since it is quite generic)

Reply:
It's created only for unit test.
I'll change the class name to containing means of test.